pkg/process/checks/process_test.go

```diff
@@ -3,6 +3,8 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
+//go:build test
```

Comment:
FYI you don't need to add the `test` build tag in test files (files ending in `_test.go`)
It's only useful for files containing code only used by tests, typically named something like `test_helpers.go`

Reply:
There is an ongoing incident about this failure (#incident-41581) so it might not fully be due to this change
Although it seems you're adding 60KiB to the iot package, most likely due to adding [those packages](https://github.com/DataDog/datadog-agent/pull/39354/s#issuecomment-3133949074) to the build